app/views/users/password_reset_reset.erb

```diff
@@ -0,0 +1,12 @@
+<link rel="stylesheet" href="../style.css" type="text/css" />
+<link href='http://fonts.googleapis.com/css?family=Open+Sans' rel='stylesheet' type='text/css'>
+
+
+<% if token? %>
+  <p>Almost there. Please enter a new password</p>
+  <form action="/users/password_reset" method="post">
```

Comment:
This form posts to the previous part of the process - where does the password actually get changed?